samples/DotNet/Microsoft.Azure.ServiceBus/BasicUwpSendReceiveSample/BasicUwpSendReceiveSample.csproj

```diff
@@ -0,0 +1,149 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Too bad the new csproj format is not possible here yet. Would remove so much noise.